loom/graphics/gfxQuadRenderer.cpp

```diff
@@ -226,7 +226,7 @@ VertexPosColorTex *QuadRenderer::getQuadVertices(TextureID texture, uint16_t num
         return NULL;
     }
 
-#ifndef LOOM_NDEBUG
+#ifdef LOOM_DEBUG

```

Comment:
Additionally, I think LOOM_NDEBUG didn't even exist, so maybe it was like a second debug switch that meant "really remove debug statements for perf".